test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
The broken test case is to handle the situation where the callback returns 0 due to reasons and the TLSEXT has already been put into the ServerHello; this specifically tests the added code that adds an empty session ticket, rather than crashing.